dev/sparktestsupport/modules.py

```diff
@@ -337,6 +337,7 @@ def __hash__(self):
         "pyspark.sql.group",
         "pyspark.sql.functions",
         "pyspark.sql.readwriter",
+        "pyspark.sql.streaming",
```

Comment:
this is pretty significant. I had no idea this file existed. I wonder if we could possibly add a check to the linter to catch new Python files appearing but not being added to this file? It's not a blocker message but an advisory one.